server/src/internalClusterTest/java/org/elasticsearch/action/IndicesRequestIT.java

```diff
@@ -516,16 +514,6 @@ public void testDeleteIndex() {
         assertSameIndices(deleteIndexRequest, TransportDeleteIndexAction.TYPE.name());
     }
 
-    public void testGetMappings() {
```

Comment:
I deleted this test because it didn't seem to make much sense to keep it when we run the action on the local node -- and it was failing.